src/retrace-server-reposync-faf

```diff
@@ -84,11 +84,11 @@ def generate_repo(db, outputdir: Path, opsys, release, arch):
 
     for name, path, db_to_update in repomdrecords:
         # Compress sqlite files with bzip2
-        if path.endswith('.sqlite'):
+        if path.suffix == '.sqlite':
             new_path = '%s.bz2' % path
             cr.compress_file(path, new_path, cr.BZ2)
-            Path(path).unlink()
-            path = new_path
+            path.unlink()
+            path = Path(new_path)
 
         record = cr.RepomdRecord(name, path)
```

Comment:
Just to make sure: Does `RepomdRecord.__init__` accept a `Path` or does it require a `str`?

It appears that it relies on the latter: https://github.com/rpm-software-management/createrepo_c/blob/fd950310a91b10c3a02c0d3ca1640652aac3ca2d/src/python/repomdrecord-py.c#L99